docs/core/extensions/localization.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Localization in .NET
+description: Learn how to use the IStringLocalizer and IStringLocalizerFactory implementations with dependency injection in your .NET workloads.
+author: IEvangelist
+ms.author: dapine
+ms.date: 08/06/2021
+---
+
+# Localization in .NET
+
+In this article, you will learn how to use the <xref:Microsoft.Extensions.Localization.IStringLocalizer%601> and <xref:Microsoft.Extensions.Localization.IStringLocalizerFactory> implementations. Localization is the process of translating an application's resources into localized versions for each culture that the application supports.
+
+All of the example source code in this article relies on the [`Microsoft.Extensions.Localization`](https://www.nuget.org/packages/microsoft.extensions.localization) and [`Microsoft.Extensions.Hosting`](https://www.nuget.org/packages/microsoft.extensions.hosting) NuGet packages. For more information on hosting, see [.NET Generic Host](generic-host.md).
+
+## Resource files
+
+The primary mechanism for isolating localizable strings is with **resource files**. A resource file is an XML file with the *.resx* file extension. Resource files are translated prior to the execution of the consuming application &mdash; in other words, they represent translated content at rest. A resource file name most commonly contains a locale identifier, and takes on the following form:
+
+**`{FullTypeName}[.Locale].resx`**
```

Comment:
The CG says to use angle brackets for placeholders, and perhaps italics too. https://review.docs.microsoft.com/en-us/help/contribute/code-in-docs?branch=master#placeholders

Reply:
Ah, good to know. Fixed